lib/diagnostic.js

```diff
@@ -32,16 +48,22 @@ const newArray = function newArray(/* length, defaultValue */) {
 // to the result of invoking defaultValue with the
 // element's index as an argument
 
+
+
+
+
 // Questions 5
 // Write a function that adds or replaces a method on an object
 // using the given propertyName
 // If the argument obj is unset, create a new object
 // The function you write should return
 // the object passed in as an argument or the object created
-const addMethod = function addMethod(/* propertyName, method, obj */) {
-
+const addMethod = function addMethod(propertyName, method, obj) {
+propertyName[method] = obj;
 };
 
+
+
```

Comment:
Please study this solution. The syntax for adding a property on an object is `obj[propertyName] = method`.
```
const addMethod = function addMethod(propertyName, method, obj) {
  if (!obj) {
    obj = {};
  }
  obj[propertyName] = method;

  return obj;
};
```